action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
   payload-file-path: # path to JSON payload to send to Slack via webhook
     description: 'path to JSON payload to send to Slack if webhook route. If not supplied, json from GitHub event will be sent instead. If payload is provided, it will take preference over this field'
     required: false
+  use-file-as-valid-json:
+    description: 'Do not try to speically parse github context, markup or other parts. Assume file content is legit'
```

Comment:
Depending on the final naming of this attribute we might want to update this to something like "Replace templated variables with the GitHub context and environment variables"